app/src/main/java/org/lineageos/twelve/services/PlaybackService.kt

```diff
@@ -51,8 +53,12 @@ class PlaybackService : MediaLibraryService(), Player.Listener, LifecycleOwner {
             mediaSession: MediaSession,
             controller: MediaSession.ControllerInfo
         ) = lifecycle.coroutineScope.future {
+            val resumptionPlaylist = mediaRepositoryTree.getResumptionPlaylist()
             MediaSession.MediaItemsWithStartPosition(
-                mediaRepositoryTree.getResumptionPlaylist(), 0, 0
+                resumptionPlaylist.mediaItemIds.map {
+                    mediaRepositoryTree.getItem(it) ?: error("Media item not found")
```

Comment:
Actually this may be wrong in case the media gets removed.
In such cases we need to offset the startIndex